packages/math-wallet/src/lib/math-wallet.ts

```diff
@@ -8,6 +8,7 @@ import {
 } from "@near-wallet-selector/core";
 
 import { InjectedMathWallet, SignedInAccount } from "./injected-math-wallet";
+import { SignedTransaction } from "near-api-js/lib/transaction";
```

Comment:
We need to avoid (where possible) importing from internal paths like this. I believe `SignedTransaction` is exported via `transactions.SignedTransaction`